.github/workflows/continuous-integration.yml

```diff
@@ -97,61 +97,29 @@ jobs:
       - name: check if translations are marked as outdated
         shell: bash
         env:
-          MASK: '---\n(.*\n)*?((outdated: |outdated_since: ).+\n)(.*\n)*?---'
           PULL_REQUEST_TAG: 'SKIP_OUTDATED_CHECK'
         run: |
-          # https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit
-          function echo_red () { echo -e "\e[0;31m$1\e[m"; }
-          function echo_green () { echo -e "\e[0;32m$1\e[m"; }
-
-          function print_error () {
-            echo "::error::You have edited some original articles (en.md), but did not outdate their translations:"
-            while read FILENAME; do
-              echo_red "* $FILENAME"
-            done <<< "$1"
-
-            echo -e "\nIf your changes DON'T NEED to be added to the translations, add $( echo_red $PULL_REQUEST_TAG ) anywhere in the description of your pull request."
-            echo -e "Otherwise, add the following to each article's front matter (https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#front-matter):\n"
-            echo_green "---"
-            echo_green "outdated: true"
-            if [[ -n "$2" ]]; then
-              echo_green "outdated_since: $2"
+          function skip_check () {
+            if ${{ github.event_name == 'pull_request' }}; then
+              ${{ contains(github.event.pull_request.body, env.PULL_REQUEST_TAG) }}
+            elif ${{ github.event_name == 'push' }}; then {
+              commits_url="${{ github.event.push.repository.commits_url }}"
+              suffix='{/sha}'
+              desired_suffix='/${{ github.sha }}/pulls'
+              pulls_url="${commits_url/$suffix/$desired_suffix}"
+              pr_body=$( curl -sS "$pulls_url" | jq -r '.[0].body' || true )
+
+              # don't fail on network errors
+              [[ -z $pr_body ]] || [[ $pr_body =~ $PULL_REQUEST_TAG ]]
```

Comment:
Is this also related to pushes? I don't agree that it should pass on a network error.

Reply:
also shouldn't the script run on every commit, not just when pr'ed? just making sure i understand this correctly

Reply:
yeah it runs on every commit, just like other checks from `continuous-integration.yaml`